xml_inscriptions/transcribed/PA.Phil.UP.UM.L.L-1033-51.xml

```diff
@@ -152,14 +155,17 @@
         <body>
             <div type="edition">
             <ab>
-                <lb n="1"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erpernae</name></persName> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <gap reason="lost" unit="character" extent="unknown"/>
+                <lb n="1"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erpernae</name></persName> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <unclear reason="lost">N(?)</unclear> <gap reason="lost" unit="character" extent="unknown"/>
                 <lb n="2"/> ossa hic sita su<supplied reason="lost">nt</supplied>.
-                <lb n="3"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arcus</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erperna</name> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <name type="cognomen">Phila<gap reason="lost" unit="character" extent="unknown"/></name></persName>
+                <lb n="3"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arcus</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erperna</name> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>ibertus</ex></expan> <name type="cognomen">Phila<unclear reason="lost">d</unclear><gap reason="lost" unit="character" extent="unknown"/></name></persName>
```

Comment:
Note that ```<unclear>``` doesn't actually require ```@reason``` on it; the under-dot will already be handled.